### ISSUE TITLE
about uniform loss

### ISSUE THREAD

**Issue by rsy6318** [UncleMEDM/PUGAN-pytorch#40]
In loss.py line 34, I think 'pcd' does not need to transpose because the input of fps function's size is (B,N,C).

**Comment by Taylor-Swift1** [comment]
Yes, I agree with you. Actually, it will produce new problem using transpose. After FPS, the indexes of sampling are almost 0. So, it can't evaluate the uniformity loss.

**Comment by Taylor-Swift1** [comment]
This problem has been solved. My pull request has been merged.
